app/assets/stylesheets/modules/image_form.scss

```diff
@@ -0,0 +1,17 @@
+@import "variables";
+
+form {
+
+  ul li {
+    display: inline-block;
+    font-size: 0.5em;
+    width: 20%;
+
+    .postfix {
+      background: #008080;
```

Comment:
Color literals like `#008080` should only be used in variable declarations; they should be referred to via variable everywhere else.